packages/manager/modules/pci/src/projects/project/data-processing/onboarding/onboarding.controller.js

```diff
@@ -0,0 +1,66 @@
+import reduce from 'lodash/reduce';
+import illustration from './assets/data-processing.png';
+import { GUIDES } from './onboarding.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $translate,
+    $state,
```

Comment:
We tend to avoid using `$state` in our controllers.

Instead we tends to use a `goBack` directly in the routing file.